packages/yoga/src/Card/native/EventCard/EventCard.jsx

```diff
@@ -1,126 +1,117 @@
-  ${({
-    theme: {
-      yoga: {
-        components: {
-          card: { event },
-        },
-      },
-    },
-  }) => `
-    justify-content: center;
-    align-items: center;
-
-    width: 56px;
-
-    border-top-left-radius: ${event.date.radius}px;
-    border-bottom-left-radius: ${event.date.radius}px;
-    background-color: ${event.date.backgroundColor};
+  width: ${small ? yoga.spacing.xxxlarge : 280}px;
+  height: 104px;
+  border-radius: ${yoga.radii.regular};
+  padding: ${yoga.spacing.zero};
   `}
 `;
 
-const EventInfo = styled.View`
-  ${({
-    theme: {
-      yoga: {
-        components: {
-          card: { event },
-        },
-      },
-    },
-  }) => `
-    flex: 1;
-    padding: ${event.info.padding.top}px ${event.info.padding.right}px
-      ${event.info.padding.bottom}px ${event.info.padding.left}px;
+const DateInfo = styled(Box)`
+  ${({ theme: { yoga } }) => `
+  justify-content: center;
+  align-items: center;
+  width: ${yoga.spacing.xxxlarge}px;
   `}
 `;
 
-const DayOfWeek = styled(Text.Small)`
-  ${({
-    theme: {
-      yoga: {
-        components: {
-          card: { event },
-        },
-      },
-    },
-  }) => `
-    margin-bottom: ${event.date.dayOfWeek.marginBottom}px;
-
-    font-weight: ${event.date.dayOfWeek.fontWeight};
-  `}
+const EventInfo = styled(Box)`
+  flex: 1;
 `;
 
-const Month = styled(Text.Small)`
-  ${({
-    theme: {
-      yoga: {
-        components: {
-          card: { event },
-        },
-      },
-    },
-  }) => `
-    font-weight: ${event.date.month.fontWeight};
+const Top = styled(Box)`
+  ${({ theme: { yoga } }) => `
+  justify-content: center;
+  align-items: center;
+  height: ${yoga.spacing.small}px;
+  margin-bottom: ${yoga.spacing.xxxsmall / 2}px;
   `}
 `;
 
 const Name = styled(Text.Medium)`
-  ${({
-    theme: {
-      yoga: {
-        components: {
-          card: { event },
-        },
-      },
-    },
-  }) => `
-    margin-bottom: ${event.info.name.marginBottom}px;
-
-    font-size: ${event.info.name.fontSize}px;
+  ${({ theme: { yoga } }) => `
+    margin-bottom: ${yoga.spacing.xxxsmall}px;
+    line-height: ${yoga.lineHeights.small}
   `}
 `;
 
-const Place = styled(Text.Small)`
-  ${({
-    theme: {
-      yoga: {
-        components: {
-          card: { event },
-        },
-      },
-    },
-  }) => `
+const Place = styled(Text.Tiny)`
+  ${({ theme: { yoga } }) => `
     width: 180px;
-    margin-bottom: ${event.info.place.marginBottom}px;
+    margin-bottom: ${yoga.spacing.xsmall}px;
+  `}
+`;
 
-    color: ${event.info.place.color};
+const getIndicatorColor = (active, hasEvent, colors) => {
+  if (active) return colors.white;
+
+  return hasEvent ? colors.primary : colors.light;
+};
+
+const Indicator = styled(Box)`
+  ${({ theme: { yoga }, active, event: hasEvent }) => `
+    width: ${yoga.spacing.xxsmall}px;
+    height: ${yoga.spacing.xxsmall}px;
+    background-color: ${getIndicatorColor(active, hasEvent, yoga.colors)};
   `}
 `;
```

Comment:
Since the Indicator is kinda Box, we could set the bgColor as prop 
```suggestion
const getIndicatorColor = (active, hasEvent) => {
  if (active) return 'white';

  return hasEvent ? 'primary' : 'light';
};

const Indicator = styled(Box)`
  ${({ theme: { yoga }, active, event: hasEvent }) => `
    width: ${yoga.spacing.xxsmall}px;
    height: ${yoga.spacing.xxsmall}px;
  `}
`;

<Indicator 
  borderRadius="small" 
  active={active} 
  event={event} 
  bgColor={getIndicatorColor(active, Boolean(event))}
/>
```

What do you think?